assessments/projects/jaeger/self-assessment.md

```diff
@@ -0,0 +1,317 @@
+
+### Known Issues Over Time. 
+
+#### Miscellaneous Issues 
+* The Jaeger architecture relies on a central collector to receive and store trace reports from agents and microservices. The collector exposes HTTP endpoints like /api/traces for trace data submission, lacking authentication. This vulnerability opens the possibility of a Server-Side Request Forgery (SSRF) attack, enabling a compromised microservice to submit malicious trace data to the collector. Implementing token-based access enhances collector security by restricting entry to authenticated agents.
+
+### [CII Best Practices](https://www.coreinfrastructure.org/programs/best-practices-program/).
+
+* The Jaeger project has achieved passing level criteria under Open Source Security Foundation Best Practices and is in the process of obtaining the silver badge.
+https://www.bestpractices.dev/en/projects/1273 
+
+### Case Studies. 
+
+#### Ticketmaster: 
+* Consisting of over 300 microservices, the company benefits from the Jaeger project, utilizing the system to track over 100 million transactions a day. 
+* As the company scaled, they quickly learned that logging was insufficient and bulky. It helped uncomplicate the log aggregation process which generated terabytes of data daily. Jaeger simplified the debugging process by providing visibility into critical areas. – where developers could find root causes of issues and save time doing so.
+* Ticketmaster uses the Directed Acyclic Graph (DAG) view to see request flow. 
+* They also take advantage of Jaeger’s powerful adaptive sampling feature which reduces overhead without reducing visibility into systems.
+#### Grafana Labs: 
+* They needed a solution to trace request issues end-to-end. Grafana sometimes dealt with high request latency and opaque request paths. 
+* Jaeger allowed engineers to sort requests by duration with the Jaeger UI, which identified slow problematic requests and allowed for more granular troubleshooting. 
+* One feature they particularly like is the contextual logging capability. 
+
+
+### Related Projects / Vendors. 
+
+**ZipKin** was an earlier open source distributed tracing system which is used to help users monitor and troubleshoot microservice-based architectures. Both Zipkin and Jaeger aim to provide visibility into the flow of requests and responses across various services in a distributed system. While Zipkin has been around longer, Jaeger is known for its scalability to handle tracing in large and complex microservices and displaying those traces on the Web UI. Jaeger also has backward compatibility with Zipkin to help users transition from Zipkin to Jaeger. While Zipkin has been around longer than Jaeger, Jaeger has the benefit of being a part of Cloud Native Computing Foundation(CNCF), supporting containers, kubernetes and OpenTracing. To conclude, the decision between the two comes down to preference, supported languages and whatever is compatible with your existing tech stack.
+
+### Action Items.
```

Comment:
Please move the action items to another file within same folder, and link it in the appendix.